src/controllers/monster.js

```diff
@@ -602,6 +603,95 @@ class Monster extends Controller {
 						data.futureEventTrigger = event.reason
 					}
 
+					//sendscout by bigfun
+					data.scoutResult = ''
+					sendscout: if ((data.iv === -1 && this.config.scouts?.scoutURL && (data.seenType != 'wild' || (data.seenType =='wild' && this.config.scouts?.scoutWild)) && !this.config.scouts?.scoutBlackList.includes(data.pokemonId)) && (this.config.scouts?.limitScoutRarity <= data.rarityGroup || data.rarityGroup == -1 && this.config.scouts?.limitScoutRarity < 2)) {
+						this.log.info(`[SCOUT] Readying ${monster.name}(${data.pokemonId}) rarityGroup: ${data.rarityGroup} - limitScoutRarity: ${this.config.scouts?.limitScoutRarity} - Clusters: ${(this.config.scouts?.scoutClusters && data.seenType == 'cell')}.`)
+
+						const coords = []
+						coords.push([data.latitude,data.longitude])
+						if (this.config.scouts?.scoutClusters && data.seenType == 'cell') {  //Cluster Scans w/ better math thanks to @RPOT
+							data.scoutResult = 'Cluster '
```

Comment:
Overall this looks like lint has not been applied.  Please 'npm run lint' and see

Reply:
I am unable to `npm run lint` in my windows environment:
```
> npm run lint

> poracle@4.8.0 lint
> ./node_modules/.bin/eslint ./src --fix

'.' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
update: linted